source/hardware/controllers/numark_dj2go2_touch.rst

```diff
@@ -0,0 +1,70 @@
+.. _numark-dj2go2-touch:
+
+Numark DJ2GO2 Touch
+===================
+
+.. sectionauthor::
+   tandy1000 <https://github.com/tandy-1000>
+
+The Numark DJ2GO2 Touch is a USB :term:`MIDI` controller with an integrated audio interface.
+It is the successor to the DJ2GO2, and features touch capable wheels.
+
+-  `Manufacturer’s product page <https://www.numark.com/product/dj2go2-touch>`__
+-  `Mapping Forum Thread <https://mixxx.discourse.group/t/a-complete-numark-dj2go2-touch-mapping/22616>`__
+
+.. versionadded:: 2.3.1
+
+Controller Mapping
+------------------
+
+Everything in this mapping is standard except:
+
+* the Level knobs have been remapped to a filter effect,
+* the browse encoder has two added features:
+
+  * it can be used to preview tracks when clicking on a track in a tracklist, scrolling the encoder allows seeking the previewed track,
+  
+  * clicking when a track is being previewed will stop the preview, allowing you to scroll the tracklist again.
```

Comment:
Please add the browse knob to the table with the "Mixer" section controls (maybe rename it to "Mixer/Library"), then we can remove this section entirely,